app/src/main/java/com/github/se/project/ui/map/LocationPickerScreen.kt

```diff
@@ -70,7 +71,7 @@ fun MapPickerBox(
             onClick = {
               onLocationSelected(selectedPosition.latitude to selectedPosition.longitude)
             },
-            modifier = Modifier.fillMaxWidth(),
+            modifier = Modifier.testTag("confirmLocation").fillMaxWidth(),
             enabled = selectedPosition.latitude != 0.0 || selectedPosition.longitude != 0.0,
```

Comment:
An unintended consequence of this is that it technically prevents a lesson from being at 0,0. Thankfully, that corresponds to a spot in the middle of the Atlantic Ocean, so we should be fine. Still, we should be careful making that the default, since it's technically a valid location.